l10n_th_account_report/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 
 {
     'name': 'Thailand Localization - Accounting Reports',
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.0.1',
```

Comment:
```suggestion
    'version': '12.0.1.1.0',
```
![image](https://user-images.githubusercontent.com/1973598/61424376-6f7be300-a93d-11e9-8183-d94e9996024f.png)